File: Test 1 - Pig latin - word (simple)/1.js
function pigLatin(word) {
    const vowels = "aeiouyAEIOUY";
    
    if (!word) {
        return "";
    }

    for (let i = 0; i < word.length; i++) {
        if (vowels.includes(word[i])) {
            return word.slice(i) + word.slice(0, i) + "ay";
        }
    }

    return word + "ay";
}

File: Test 1 - Pig latin - word (simple)/1.py
def pig_latin(word):
    vowels = "aeiouyAEIOUY"
    
    if not word:
        return ""


    for i, letter in enumerate(word):
        if letter in vowels:
            return word[i:] + word[:i] + "ay"

    return word + "ay"
       

print(pig_latin("stop"))
print(pig_latin("no"))
print(pig_latin("people"))
print(pig_latin("bubble"))
print(pig_latin("under"))
print(pig_latin("admitted"))
print(pig_latin("away"))
